spec.html

```diff
@@ -4697,7 +4697,7 @@ <h1>CreateListFromArrayLike ( _obj_ [ , _elementTypes_ ] )</h1>
           1. Let _next_ be ? Get(_obj_, _indexName_).
           1. If Type(_next_) is not an element of _elementTypes_, throw a *TypeError* exception.
           1. Append _next_ as the last element of _list_.
-          1. Set _index_ to _index_ + 1.
+          1. Increase _index_ by 1.
```

Comment:
0366834da60a5f311c34fec3ddd2c8d686f52d8a: Can you elaborate on "seems more popular"? I kind of feel like "Set x to x + 1" is more explicit.

Reply:
> Can you elaborate on "seems more popular"?

The status quo spec has roughly these counts:
  - 48 `Increase _x_ by 1.` (plus a `by 2` and a `by 5`)
  - 14 `Increment _x_ by 1.` (plus a couple `by 2`, plus one with an implied `by 1`)
  - 15 `Let _x_ be _x_ + 1.`
  - 11 `Set _x_ to _x_ + 1.`

> I kind of feel like "Set x to x + 1" is more explicit.

I don't know, I might argue that `Increase _x_ by 1` is more explicit. I think it makes it *slightly* easier to recognize the 'loop variable' in a `Repeat` command.

But I don't think I care. If the editors decide to deprecate Increase/Increment/Decrease/Decrement, I'll withdraw this commit and submit one to convert them all to `Set _x_ to _x_ +/- 1`.

Reply:
I totally agree we should pick one; to me either "increment" or "{S,L}et x to x + N" is better. As for whether it's `let` or `set`, that would depend on the context.

Reply:
> "increment"

(Personally, if the choice were only between `increment` and `increase`, I too would prefer `increment`, as it sounds more "integer-y" to me.)

> As for whether it's let or set, that would depend on the context.

I don't think so. If `_x_` appears on the RHS, it must already be in scope (must have been declared earlier), so changing its value would call for`Set`.